valkyrie/lib/valkyrie/specs/shared_specs/queries.rb

```diff
@@ -8,6 +8,7 @@ class CustomResource < Valkyrie::Resource
       attribute :title
       attribute :member_ids
       attribute :a_member_of
+      attribute :checksum, Valkyrie::Types::Hash
```

Comment:
I thought we were storing `multihash` so we wouldn't need to put a hash in?

Reply:
Good call 👍  Thought this would be a simpler implementation to try at first, but working on the `multihash` functionality this afternoon.

Reply:
@tpendragon I'm trying to store a `multihash` in `attributes :checksum` (without an explicit Type)
```
let(:multihash_sha_256) { Multihashes.encode(Digest::SHA256.digest('secret'), 'sha2-256') }
let!(:resource1) { persister.save(resource: resource_class.new(checksum: [multihash_sha_256])) }
```

which works in the memory persister but I'm getting character encoding errors in the Fedora::QueryService coming from the `property_converter` in `Valkyrie::Persistence::Fedora::Persister::ModelConverter` because `multihashes` encodes to an ASCII-8BIT string which I think the `property_converter` is having trouble converting UTF-8:
```
Encoding::UndefinedConversionError:
       "\xB8" from ASCII-8BIT to UTF-8
```

I'm guessing I'll have to circle back around to this on Monday.

Reply:
Gotta love string encoding..